addons/website_sale/tests/test_sale_process.py

```diff
@@ -405,7 +405,7 @@ def test_07_change_fiscal_position(self):
             'list_price': 100,
             'website_published': True,
             'sale_ok': True,
-            'taxes_id': [tax_10_incl.id]
+            'taxes_id': [Command.set(tax_10_incl.ids)],
```

Comment:
What exactly is making this change necessary ?
Either it wasn't working before, or I don't see what change you do that makes it not work anymore :thinking:.

Reply:
It made the older implementation cleaner and I decided to leave it in the commit even after the new implementation not needing it. Should I change it back?

Reply:
I don't mind, but I was curious why this change was done ;)